src/disc-golf-events/DiscGolfEventsComponent.js

```diff
@@ -0,0 +1,58 @@
+import { useEffect, useState, useRef } from "react";
+import Table from "@mui/material/Table";
+import TableBody from "@mui/material/TableBody";
+import TableCell from "@mui/material/TableCell";
+import TableContainer from "@mui/material/TableContainer";
+import TableHead from "@mui/material/TableHead";
+import TableRow from "@mui/material/TableRow";
+import Paper from "@mui/material/Paper";
+
+const DiscGolfEventsComponent = () => {
+
+      const [discGolfEvents, setDiscGolfEvents] = useState([]);
+
+      useEffect(() => {
+        fetchEvents();
+      }, []);
+
+      const fetchEvents = () => {
+        console.log("Fetching now");
+        fetch("http://localhost:8080/events")
+          .then((response) => response.json())
+          .then((data) => setDiscGolfEvents(data));
```

Comment:
parenthesis around parameters could be removed (I'm not sure, please check)
```suggestion
          .then(response => response.json())
          .then(data => setDiscGolfEvents(data));
```